src/plugins/friendInvites.ts

```diff
@@ -35,38 +35,61 @@ export default definePlugin({
             name: "create friend invite",
             description: "Generates a friend invite link.",
             inputType: ApplicationCommandInputType.BOT,
-            execute: async (_, ctx) => {
-                if (!UserStore.getCurrentUser().phone)
+            options: [{
+                name: "Uses",
+                displayName: "Uses",
+                description: "How many uses?",
+                displayDescription: "How many uses?",
+                choices: [{ "label": "1", "name": "1", "value": "1" }, { "label": "5", "name": "5", "value": "5" }],
+                required: true,
```

Comment:
it'd be better UX to add a default value
```suggestion
                required: false,
```